docs/partial_source/deep_dive/ivy_frontends.rst

```diff
@@ -199,18 +205,51 @@ Ivy does not support the unique naming of individual operations, and so we omit
 .. code-block:: python
 
     # in ivy/functional/frontends/tensorflow/math.py
+    @to_ivy_arrays_and_back
     def tan(x, name=None):
         return ivy.tan(x)
 
 Likewise, :code:`tan` is also placed under :mod:`math`.
 By referring to the `tf.math.tan`_ documentation, we add the same arguments, and simply wrap :func:`ivy.tan` in this case.
 Again, we do not support the :code:`name` argument for the reasons outlined above.
 
+**NOTE**
+
+Many of the functions in the :mod:`tf.raw_ops` module have identical behaviuor to the functions that are generally in the TensorFlow namespace e.g :func:`tf.argmax`, with exception that the functions in the function in the :mod:`tf.raw_ops` module are specified to have key-word only arguments and in some case there is a naming difference of arguments.
+In order to tackle these variations in behaviour, the :code:`map_raw_ops_alias` decorator was designed to wrap the functions that exist in the TensorFlow namespace, thus reducing unnecessary re-implementations.
+
+.. code-block:: python
+    
+    # in ivy/functional/frontends/tensorflow/math.py
+    @to_ivy_arrays_and_back
+    def argmax(input, axis, output_type=None, name=None):
+        if output_type in ["uint16", "int16", "int32", "int64"]:
+            return ivy.astype(ivy.argmax(input, axis=axis), output_type)
+        else:
+            return ivy.astype(ivy.argmax(input, axis=axis), "int64")
+
+This function :func:`argmax` is implemented in the :mod:`tf.math` module of the TensoFlow framework, there exists identical function in the :mod:`tf.raw_ops` module implemented as :func:`ArgMax`. Both the functions have identical behaviour except that all arguments are passed as key-word only for :func:`tf.raw_ops.ArgMax` and in some corner cases, arguments are renamed where :code:`dimension` argument replaces :code:`axis` in case of the :func:`tf.math.argmax` function.
```

Comment:
there exists identical - > there exists an identical

Reply:
Both the functions have identical behaviour except that all -> both the functions have identical behaviour except for the fact that all

Reply:
arguments are renamed where :code:`dimension` argument replaces :code:`axis` in case of the :func:`tf.math.argmax` function. -> such as the case in :func:`tf.math.argmax`, the :code:`dimension` argument replaces :code:`axis`.